### DISCUSSION TITLE
why need installed fontconfig ?

### DISCUSSION THREAD

**Discussion by meihuanyu** [samizdatco/skia-canvas#86] [Q&A] [answered]
instead of linked into the library.

**Comment by samizdatco** [comment, marked as answer]
As of version 3.0 there's a static version of libfontconfig embedded in Linux versions of the library. It still looks for config files in the usual places (`/etc/fonts`, `~/.config/fontconfig`, etc.) via its built-in [`fonts.conf`](https://github.com/samizdatco/skia-canvas/blob/main/lib/fonts/fonts.conf) so you should be able to configure it as you see fit.
